config/default.yml

```diff
@@ -110,6 +110,12 @@ RSpec:
       - subject
       - subject!
 
+Metrics/BlockLength:
```

Comment:
Unfortunately enough, this doesn't seem to work. I've checked by using this branch on a project and removing the explicit `Exclude` for specs from its `.rubocop.yml`.

This seems to work like this for [`rubocop-rails`'s `Style/AndOr` override](https://github.com/rubocop/rubocop-rails/blob/944394b9a45c82137cd9af8852ef43b3cd472265/config/default.yml#L981).

I can think of [`inherit_mode`](https://docs.rubocop.org/rubocop/1.27/configuration.html#merging-arrays-using-inherit_mode), and that the user project's configuration `Exclude` overrides the one from `config/default.yml`.